### PR TITLE
inspector_components_transfer

### PR DESCRIPTION
**Pull request by vpereverzev**
All general QML components were moved to the separate plugin.

So now we can use any general QML component within the app using the UiComponents plugin

изображение

### CODE REVIEW

File: inspectors/CMakeLists.txt [hunk 1]
```diff
@@ -73,4 +73,4 @@ xcode_pch(inspectors all)
 # Use MSVC pre-compiled headers
 vstudio_pch( inspectors )
 
-target_link_libraries(inspectors global global_old)
+target_link_libraries(inspectors global global_old ui uicomponents)
```
Comment (igorkorsukov) [inline inspectors/CMakeLists.txt:76]: Why here link  `ui`, `uicomponents` ?

Comment (vpereverzev) [reply inspectors/CMakeLists.txt:76]: forgot to delete, thanks

File: framework/ui/iconcodes.h [hunk 2]
```diff
@@ -58,17 +55,17 @@ class IconCode
...
         METRONOME = 0xEF20,
         TUNING_FORK = 0xEF21,
-        FILE_NEW = 0xEF22,
-        FILE_OPEN = 0xEF23,
-        FILE_SHARE = 0xEF24,
-        FILE_CLOUD = 0xEF25,
+        NEW_FILE = 0xEF22,
+        OPEN_FILE = 0xEF23,
+        SHARE_FILE = 0xEF24,
+        CLOUD_FILE = 0xEF25,
```
Comment (igorkorsukov) [inline framework/ui/iconcodes.h:61]: Where do the names come from?

File: framework/ui/iconcodes.h [hunk 3]
```diff
@@ -58,17 +55,17 @@ class IconCode
...
         REWIND = 0xEF26,
         MIXER = 0xEF27,
         CONFIGURE = 0xEF28,
         SAVE = 0xEF29,
         PLUS = 0xEF2A,
         MINUS = 0xEF2B,
-        DELETE = 0xEF2C,
+        DELETE_TANK = 0xEF2C,
```
Comment (igorkorsukov) [inline framework/ui/iconcodes.h:68]: If the name has changed, then you can remove above `undef`

File: framework/ui/iconcodes.h (hunk 2 above)
Comment (vpereverzev) [reply framework/ui/iconcodes.h:61]: They came from the Martin's icon font

Comment (igorkorsukov) [reply framework/ui/iconcodes.h:61]: I think you need to add comments, what kind of codes are they, where they come from, how to add another one, and so on.

**Pull request merged**